### PR TITLE
Fix RBAC issues for admin roles

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

This will be fixing the issues in the following roles
- Application view
- Application edit
- Userstore view
- Userstore edit

### Related Issues
- https://github.com/wso2/product-is/issues/18632

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/forms/provisioning-configuration-form.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020-2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/forms/provisioning-configuration-form.tsx:2]:
```suggestion
 * Copyright (c) 2020-2024, WSO2 LLC. (https://www.wso2.com).
```

File: apps/console/src/features/applications/components/settings/provisioning/inbound-provisioning-configuration.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2020-2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (jeradrutnam) [inline apps/console/src/features/applications/components/settings/provisioning/inbound-provisioning-configuration.tsx:2]:
```suggestion
 * Copyright (c) 2020-2024, WSO2 LLC. (https://www.wso2.com).
```

**Review by jeradrutnam — APPROVED** [review]

**Comment by jeradrutnam** [comment]
License header needs to update on all the changed files

File: apps/console/src/features/applications/components/forms/provisioning-configuration-form.tsx (hunk 1 above)
Comment (DonOmalVindula) [reply apps/console/src/features/applications/components/forms/provisioning-configuration-form.tsx]: Fixed with ea240a7599d73a2442fb8f5a85c7b61408b3a9ff

**Comment by DonOmalVindula** [comment]
> License header needs to update on all the changed files

Fixed with ea240a7599d73a2442fb8f5a85c7b61408b3a9ff

**Review by brionmario — APPROVED** [review]

**Pull request merged**